### PR TITLE
Update README for CT log 2022 key

### PR DESCRIPTION
**Pull request by Hayden-IO**
Signed-off-by: Hayden B <hblauzvern@google.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,8 @@ The current published repository metadata lives in the [repository](/repository/
...
 | [rekor.0.pub](repository/repository/targets/rekor.0.pub)        |  This is a dupe of `rekor.pub` and will be removed in the next root-signing event. | 
-| [ctfe.pub](repository/repository/targets/ctfe.pub)        |  Certificate Transparency log key that is used for certificates issued by Fulcio and used to verify signed certificate timestamps (SCTs) for inclusion into the log. | 
+| [ctfe.pub](repository/repository/targets/ctfe.pub)        |  Certificate Transparency log key for the first log shard (`ctfe.sigstore.dev/test`), that is used for certificates issued by Fulcio and used to verify signed certificate timestamps (SCTs) for inclusion into the log. | 
+| [ctlog2022.pub](repository/repository/targets/ctlog2022.pub)        |  Certificate Transparency log key for the 2022 log shard (`ctfe.sigstore.dev/2022`), that is used for certificates issued by Fulcio and used to verify signed certificate timestamps (SCTs) for inclusion into the log. |
```
Comment (Hayden-IO) [inline README.md]: This path doesn't exist yet, but it will soon enough

**Review by dlorenc — DISMISSED** [review]

**Comment by asraa** [comment]
Will merge after we stage the new root

**Comment by Hayden-IO** [comment]
This should be good to merge, just need approval

**Review by cpanato — APPROVED** [review]

**Review by asraa — APPROVED** [review]
Thank you!

**Pull request merged**